Doppler.HtmlEditorApi/Storage.DapperProvider/Repository.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Threading.Tasks;
+using Dapper;
+
+namespace Doppler.HtmlEditorApi.Storage.DapperProvider;
+
+public class Repository : IRepository
+{
+    // TODO: add types to represent Dapper queries results.
+    // Because dynamic Dapper results do nowork with Moq.Dapper.
+
+    private const int EDITOR_TYPE_MSEDITOR = 4;
+    private const int EDITOR_TYPE_UNLAYER = 5;
+
+    private readonly IDatabaseConnectionFactory _connectionFactory;
+    public Repository(IDatabaseConnectionFactory connectionFactory)
+    {
+        _connectionFactory = connectionFactory;
+    }
+
+    public async Task<ContentData> GetCampaignModel(string accountName, int campaignId)
+    {
+        using (var connection = await _connectionFactory.GetConnection())
+        {
+            var databaseQuery = @"
```

Comment:
just a detail, we can use a constant here
```suggestion
            const string databaseQuery = @"
```

Reply:
Done here: https://github.com/FromDoppler/doppler-html-editor-api/pull/44/files#diff-670f8ae2f5aff6be803837e52585a38c81e9738fa7b599d982d52a2836c64f36R9